Tests/NumLib/TestODEInt.cpp

```diff
@@ -279,10 +279,13 @@ class NumLibODEIntTyped : public ::testing::Test
 
 TYPED_TEST_CASE(NumLibODEIntTyped, TestCases);
 
+// Temporarily disabled for PETSc issue #
+#ifndef USE_PETSC
```

Comment:
Minor: It would be better to disabler unit tests like this:
```
#ifndef USE_PETSC
TYPED_TEST(NumLibODEIntTyped, T1)
#else
TYPED_TEST(NumLibODEIntTyped, DISABLED_T1)
#endif
```